### PR TITLE
Added keep existing configuration option to all combos

### PR DESCRIPTION
**Pull request by SachinS10-lab**
- Added keep existing configuration option to all combos.
- Added VscCircleLargeFilled icon for keep existing configuration option selection.

### CODE REVIEW

File: webviews/src/App.tsx [hunk 1]
```diff
@@ -11,10 +11,12 @@ function App() {
...
     { label: 'granite-code:8b', value: 'granite-code:8b', info: '4.6 GB' },
     { label: 'granite-code:20b', value: 'granite-code:20b', info: '12 GB' },
-    { label: 'granite-code:34b', value: 'granite-code:34b', info: '19 GB' }
+    { label: 'granite-code:34b', value: 'granite-code:34b', info: '19 GB' },
+    { label: 'Keep existing configuration', value: 'Keep existing configuration', info: '' }
   ];
   const embeddingsOptions: ModelOption[] = [
-    { label: 'nomic-embed-text', value: 'nomic-embed-text:latest', info: '274 MB' }
+    { label: 'nomic-embed-text', value: 'nomic-embed-text:latest', info: '274 MB' },
+    { label: 'Keep existing configuration', value: 'Keep existing configuration', info: '' }
```
Comment (fbricon) [inline webviews/src/App.tsx:19]: can we have a null value instead? That'd avoid having to touch the onChange handlers

**Comment by fbricon** [comment]
as expressed in #9, if no granite code is selected for either choice, the Install button should be disabled

File: webviews/src/App.tsx (hunk 1 above)
Comment (SachinS10-lab) [reply webviews/src/App.tsx:19]: yes we can. But for onChange handler , we need to check 'Keep existing configuration' option and not to pass value to the config.JSON file.

**Comment by SachinS10-lab** [comment]
> as expressed in #9, if no granite code is selected for either choice, the Install button should be disabled

I will add that. Now I reset my main's HEAD to previous commit and add mentioned changes to my feature branch and rise PR from that.

**Pull request closed without merging**